src/state/download/actions.js

```diff
@@ -26,7 +26,7 @@ export const updateDownloadDataSet = data => ({
   type: 'DOWNLOAD_DATASET_UPDATE',
   data,
   persist: {
-    dataSetId: data.dataSetId,
+    dataSetId: data.dataSetId || data.id,
```

Comment:
Thanks for sending this, it fixes the issue! 🔥 

Is it possible to find where this is called and pass the `dataSetId` there? It would be nice if we could have a single parameter instead of using both.

Reply:
Yeah I can look into it more, I have a hunch that its coming back from the API inconsistently. So I'm not sure at what level you want to sanitize the id. I left another comment on the issue thread that helps reveal when the value changes. 

Do you want to merge this or wait and address the inconsistency before it gets to the downloads reducer or maybe at the reducer we can check if dataSetId is empty and try id... thoughts?

Reply:
hmm I think the problem might be related to how we parse the api response, since the api always returns `id` and we use `dataSetId` in the code.

Yeah let's get this merged and we can find a better solution later.